share/getter.go

```diff
@@ -35,6 +39,8 @@ type Getter interface {
 
 	// GetSharesByNamespace gets all shares from an EDS within the given namespace.
 	// Shares are returned in a row-by-row order if the namespace spans multiple rows.
+	// If data for the requested root does not include any shares from the given namespace
+	// ErrNamespaceNotFound will be returned along with non-inclusion proofs if there are any.
```

Comment:
We should still assess the choice of returning value and the error on the error case:
* It's not idiomatic. Even io.EOF is returned after we read out everything.
* The go-jsonrpc may enforce idiomatic behavior and return either rather than both.